### ISSUE TITLE
allow `tag` to delete tags via `-tagName` not only `+-tagName`

### ISSUE THREAD

**Issue by zombiefungus** [pubs/pubs#206]
Or at least document the fact that the '+' is needed, otherwise is quite hard to know how to remove a tag

**Comment by benureau** [comment]
Thanks for the issue @zombiefungus. And, indeed. That's not intended. `-tagName` is parsed as a command line option, rather than a tag declaration. I will look if we can work around this. If not, I will indeed document it.

**Comment by zamlz** [comment]
Currently tags can also be removed with `:tagName`. I proposed a workaround but I'm not sure if its any good in issue #175

**Comment by zombiefungus** [comment]
I would suggest to consider the + and - anyway. This is how is done in another tool I use [taskwarrior](https://taskwarrior.org/docs/examples.html#tags) I understand it could be a significant change but having tags separated by space and using the +/- format would solve the problem of having hyphens in tag names

What do you think?

**Comment by benureau** [comment]
Yep, that seems interesting indeed. And the current implementation we have is obviously confusing. But as it may break users pipelines/habits, we probably need a major version released to implement this. I found a way with `parse_known_args` that I think will work on resolving this issue that's quick enough. I'll do this, and then we can discuss how to improve the tag command.

**Comment by zamlz** [comment]
Maybe we can have the operation under quotes. Like we can use `'-tagName'`.
I don't think the python argparse library would recognize that as argument
and it's still intuitive I think

**Comment by omangin** [comment]
Good points. I would still keep an unambiguous syntax to be sure there is something that works even when a tag is named as an option but that is not incompatible with enabling the trailing `-` for the other cases.
